test/test_core.py

```diff
@@ -4131,6 +4131,7 @@ def test_dylink_basics_no_modify(self):
     if self.get_setting('MEMORY64') == 2:
       self.skipTest('MEMORY64=2 always requires module re-writing')
     self.set_setting('WASM_BIGINT')
+    self.set_setting('STACK_OVERFLOW_CHECK', 1)
```

Comment:
Are these just to add more test coverage? Or is it necessary for some reason? Please add a comment.